### PR TITLE
tegra: enable systemd-boot support

### PR DESCRIPTION
**Pull request by igoropaniuk**
Enable systemd-boot support by default.

### CODE REVIEW

**Comment by igoropaniuk** [comment]
Depends on https://github.com/OE4T/meta-tegra/pull/1687

File: meta-lmp-bsp/conf/machine/include/lmp-machine-custom.inc [hunk 1]
```diff
@@ -772,22 +772,28 @@ DISTRO_FEATURES:append:tegra = " opengl"
...
 PREFERRED_PROVIDER_virtual/kernel:tegra ?= "linux-tegra"
 IMAGE_TEGRAFLASH_FS_TYPE:sota:tegra = "${@bb.utils.contains('TEGRA_EXTERNAL_ROOTFS_WIC', '1', 'wic.gz', 'ota-ext4', d)}"
+IMAGE_TEGRAFLASH_ESPIMG:sota:tegra = "${IMGDEPLOYDIR}/${IMAGE_LINK_NAME}.ota-esp"
 IMAGE_FSTYPES:remove:tegra = "${@bb.utils.contains('TEGRA_EXTERNAL_ROOTFS_WIC', '1', '', 'wic wic.gz wic.bmap', d)}"
+TEGRA_ESP_IMAGE:sota:tegra = ""
 TEGRA_INITRAMFS_FSTYPES:tegra = ""
 INITRD_IMAGE:tegra = ""
 IMAGE_TEGRAFLASH_KERNEL:tegra = "${DEPLOY_DIR_IMAGE}/${KERNEL_IMAGETYPE}"
 WKS_FILE:tegra:sota ?= "efidisk-sota.wks.in"
+IMAGE_EFI_BOOT_FILES:sota:tegra ?= "${@make_efi_dtb_boot_files(d)}"
```
Comment (ricardosalveti) [inline meta-lmp-bsp/conf/machine/include/lmp-machine-custom.inc:782]: Wonder if we need the dtb boot files here.

Comment (igoropaniuk) [reply meta-lmp-bsp/conf/machine/include/lmp-machine-custom.inc:782]: not really, I've removed the line

**Review by ricardosalveti — APPROVED** [review]
LGTM

**Review by MrCry0 — APPROVED** [review]

**Comment by igoropaniuk** [comment]
@quaresmajose we can merge this, all needed changes in meta-tegra are in

**Pull request merged**